src/sfctl/custom_cluster.py

```diff
@@ -93,10 +102,28 @@ def select(endpoint, cert=None, key=None, pem=None, ca=None,
         # Make sure basic GET request succeeds
         rest_client.send(rest_client.get('/')).raise_for_status()
 
+        # Get cluster version
+        cluster_code_versions = rest_client.send(
+            rest_client.get('/$/GetProvisionedCodeVersions?api-version=3.0')
+        ).json()
+
+    #pylint: disable-msg=line-too-long
+    print('Cluster      : ' + endpoint)
+    print('Code Versions: { ' + ", ".join(ver['CodeVersion'] for ver in cluster_code_versions) + " }")
+
+    for version in cluster_code_versions:
```

Comment:
This will print a meaningless message for anyone who previously had a 5.7 cluster upgraded to 6.0. I don't think this is the right mechanism to validate which version the cluster is currently on. IMO the best option would be to query the CM to respond with the current cluster version. I believe this API does not exist, so there would need to be a new feature request to add this to the Service Fabric runtime. Another hacky option is to check what the majority of up nodes report their code version is. Although this won't work in the case of upgrades.